src/components/FieldMultiSelect/index.tsx

```diff
@@ -0,0 +1,94 @@
+import { useEffect, useState } from 'react';
```

Comment:
The export of this component is missing in the components/index.ts file ;)

Reply:
@ivan-dalmet I use addons to generate it. And I see many component not exported. 

You want I push all export in this MR or another MR ?

![image](https://user-images.githubusercontent.com/4147037/155684329-08acb017-1f66-4918-bb0d-cdf08d1fd456.png)

Reply:
Do it in this one please :)

Reply:
Done :+1: